scripts/towncrier-draft-to-file.py

```diff
@@ -0,0 +1,14 @@
+from subprocess import call
+
+
+def main():
+    """
+    Platform agnostic wrapper script for towncrier.
+    Fixes the issue (#7251) where windows users are unable to natively run tox -e docs to build pytest docs.
+    """
+    with open("doc/en/_changelog_towncrier_draft.rst", "w") as draft_file:
+        return call(("towncrier", "--draft"), stdout=draft_file)
+
+
+if __name__ == "__main__":
+    exit(main())
```

Comment:
i recall that this one needs to be imported from `sys`

Reply:
Weird how come this didn't break on CI

Reply:
`exit is a helper for the interactive shell - sys.exit is intended for use in programs.` TIL. will change - not sure re: @nicoddemus  point

Reply:
I thought `exit` would exist only in the interactive shell, so I thought the test should break, but they are introduced by `site` even in programs. TIL as well. 😁

Reply:
you only need to import it if you're paranoid about siteless execution, but literally no one executes python in siteless mode